src/safariservices.cs

```diff
@@ -171,6 +176,15 @@ interface SFSafariExtensionHandling
 
 		[Export ("popoverViewController")]
 		SFSafariExtensionViewController PopoverViewController { get; }
+
+		[Mac (10, 12, 4)]
+		[Async (ResultTypeName="SFValidationResult")]
+		[Export ("validateContextMenuItemWithCommand:inPage:userInfo:validationHandler:")]
+		void ValidateContextMenuItem (string command, SFSafariPage page, [NullAllowed] NSDictionary<NSString, NSObject> userInfo, Action<bool, NSString> validationHandler);
```

Comment:
I understand the benefit of using the delegate instead, and am happy to do that, I just wanted to make sure I wasnt' missing something since you said said to remove the ResultTypeName and it wouldn't let me :).